tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAnnotations.java

```diff
@@ -272,6 +272,9 @@ public void TestCategories() {
     }
 
     public void TestUniqueness() {
+        if (logKnownIssue("CLDR-16945", "skip duplicate TestUniqueness in favor of CheckDisplayCollisions")) {
+            return;
+        }
```

Comment:
@macchiati this test seems to just duplicate CheckDisplayCollisions, so disabled for now.

Reply:
CLDR-16947 to track

Reply:
I think these were added because the CheckDisplayCollisions wasn’t catching them; so don’t delete them until we can check.

Reply:
@macchiati it's not deleted, just logKnownIssue skipped so we can make progress. Unit test failures block seeing any ConsoleCheck results, currently.